### PR TITLE
Update Prerequisites

### PR DESCRIPTION
**Pull request by linawolf**
And move them to their own page

Releases: main

### CODE REVIEW

File: Documentation/Prerequisites/Index.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+Your TYPO3 installation should be empty and have no pages, designs or third
+party extensions installed.
+
+You can follow this tutorial to install TYPO3 with DDEV and Composer:
+
+*   :ref:`Installing TYPO3 with DDEV <t3start:installation-ddev-tutorial>`.
+
+..  note:: Answer "no" when asked if you want to create a basic site.
+
+You should be able to look into the TYPO3 backend and see the following:
```
Comment (brotkrueml) [inline Documentation/Prerequisites/Index.rst:23]:
```suggestion
You should be able to log in into the TYPO3 backend and see the following:
```

File: Documentation/Prerequisites/Index.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+..  figure:: EmptyInstallation.png
+    :alt: Screenshot of a fresh TYPO3 13.4-dev installation with no page in the page tree
+
+    A fresh, empty TYPO3 installation (light mode)
+
+It is also helpful if you know the following concepts:
+
+*   Backend
```
Comment (brotkrueml) [inline Documentation/Prerequisites/Index.rst:32]: Can we link at least some of them for further reading?

**Review by brotkrueml — COMMENTED** [review]
I suggest to move at least some content from the introduction to the start page, especially, WHY a site package is recommended, like encapsulation and security (storing templates/css/js files in fileadmin is discouraged).

File: Documentation/Prerequisites/Index.rst (hunk 2 above)
Comment (linawolf) [reply Documentation/Prerequisites/Index.rst:32]: I created an issue for that as most of them have not yet properly been explained in "Getting Started": https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-SitePackage/issues/188

**Comment by linawolf** [comment]
This is a Tutorial so it should be followed step by step by beginners. The theories are neither helpful to beginners nor will they most probably be read.

The pros and cons of using a site package can be described in TYPO3 explained for advanced developers. Here we simply explain one way of how to do things.

With the recent changes it has become possible to create a site without sitepackage by having files in the site folder. Having templates in the fileadmin is completely outdated. We do not want to confuse newcomers with this.

If you feel any of the information is valuable enough you can update it and move it to a more suitable location

**Review by garvinhicking — CHANGES_REQUESTED** [review]
My oppinion: I really think this is very worthful and belongs there. A tutorial should not just be copy&paste only, but educate people what they do and why.

These are distinct pages, if people are not interested in that, they can skip it. 

If this gets removed here, it should first get added somewhere else and properly linked here.

**Comment by linawolf** [comment]
@garvinhicking, I reverted deleting the Preface and Introduction, let us discuss them separately.

Do you think the rest can be merged?

**Review by garvinhicking — APPROVED** [review]
Thanks, that makes me happy :)

**Pull request merged**